scripts/installer/windows/scripts/generate_dependency.py

```diff
@@ -57,6 +57,12 @@ def get_toml_dependencies(packages):
                 data = toml.load(file)
             extra_package_names = data.get('tool', {}).get('poetry', {}).get('dependencies', {})
             dependencies.extend(extra_package_names.keys())
+    # hard-code promptflow-evals dependency here since it's not added in promptflow setup for now
```

Comment:
Can we just do a hardcode here? why need to change the toml?

Reply:
we don't change toml here, but get dependencies in the promptflow-evals toml